pennylane/circuit_graph.py

```diff
@@ -608,3 +635,18 @@ def has_path(self, a, b):
             bool: returns ``True`` if a path exists
         """
         return nx.has_path(self._graph, a, b)
+
+    def compute_max_obs_per_wire(self):
+        """Computes the maximum number of observables defined per wire.
```

Comment:
Thanks! Went with `max_measure_per_wire`, how would that sound? Also adjusted the docstring.

Reply:
To be honest, I find the 'per wire' a bit confusing, since it sounds like it will be returning a list (with each element corresponding to a wire).

Something like 'max_simultaneous_measurements` is much more identifiable to me?

Reply:
Sounds good, changed to that! Thanks :)